analytics.js

```diff
@@ -1072,7 +1072,11 @@ class Analytics {
     }
 
     this.eventRepository.initialize(writeKey, serverUrl, options);
-    this.initializeUser();
+    if (options && options.autoAssignAnonId) {
```

Comment:
Can you think of any future use case as you suggested the below 

```
 enabled: true,
 source: "segment"
```

 fields to be nested in another object
 rather than 

```
 load(<write key>, <data plane url>, {
   autoCaptureAnonymousId: {
      enabled: true,
      source: "segment"
    }
});
```

Reply:
TBH nothing I can think of but the one I proposed allows extensibility if needed.

Reply:
Addressed